src/beachbot/control/controllerselector.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Union
+from .robotcontroller import RobotController, BoxDef
+from ..robot.robotinterface import RobotInterface
+from ..utils.controllercollection import PIDController
+from .approachdebris import ApproachDebris
```

Comment:
For all new code please use fully defined lib paths (not relative ones). 
https://app.asana.com/0/0/1208984473190453/f